### DISCUSSION TITLE
"npm dist-tag "expect inconsistency

### DISCUSSION THREAD

**Discussion by 89ao** [verdaccio/verdaccio#3613] [Q&A] [answered]
situation：
We are using verdaccio hosting a npm repo,recently I found that when I install some pkgs, It provides me different pkg versions(which are deprecated). for example :
npm install @antv/x6 @antv/x6-vue-shape 
1. our local repo provides me    +-- @antv/x6-vue-shape@3.0.0    and-- @antv/x6@3.0.0。
2. while the same time official repo provides me +-- @antv/x6-vue-shape@2.0.9    `-- @antv/x6@2.3.0

After I have some digging，I found that the reason maybe on /pkgs_name/package.json file   "dist-tags".
Checking by this way:
```
root@9bcbf910b176:/tmp/test# npm dist-tag ls @antv/x6
beta: 2.0.0-beta.5
latest: 2.3.0
root@9bcbf910b176:/tmp/test# npm dist-tag ls @antv/x6 --registry=http://mirrors.out_intranet_repo.com/npm/
beta: 2.0.6-beta.26
latest: 3.0.0
```

**So the problem become that : 
Why is package.json different from official while using verdaccio? (and how to repair this?)**

My Environment:
# verdaccio --version
4.10.0
#configuration is below（using CentOS Linux release 7.2 (Final) on kubernets）:
[root@mirrors-npm-gz-0 /tmp]# cat  ~/.config/verdaccio/config.yaml|grep -v \#|grep -v ^$
storage: /repo/npm/verdaccio/storage
http_proxy: http://xxx:3128
https_proxy: http://xxx:3128
no_proxy: localhost,127.0.0.1
plugins: ./plugins
url_prefix: /npm/
max_body_size: 10mb
listen:
   -  0.0.0.0:4873
web:
  title: Tencent-NPM
auth:
  htpasswd:
    file: ./htpasswd
    max_users: -1
uplinks:
  npmjs:
    url: https://registry.npmjs.org/
    maxage: 4m
    timeout: 10s
    agent_options:
      keepAlive: true
      maxSockets: 40
      maxFreeSockets: 10
packages:
  '@*/*':
    access: $all
    publish: $authenticated
    proxy: npmjs
  '**':
    access: $all
    publish: $authenticated
    proxy: npmjs
middlewares:
  audit:
    enabled: true
server:
  keepAliveTimeout: 60
logs:
  - {type: stdout, format: pretty, level: http}

**Comment by juanpicado** [comment, marked as answer]
verdaccio 4? `v4.10.0`?

**Reply by 89ao** [reply]
4.10.0

**Reply by juanpicado** [reply]
Sorry v4 is unmaintained  https://github.com/verdaccio/verdaccio/blob/master/SECURITY.md#supported-versions please upgrade or try to reproduce scenario with latest versions.
